protocol/chainlib/chain_fetcher.go

```diff
@@ -78,9 +78,8 @@ func (cf *ChainFetcher) Validate(ctx context.Context) error {
 				}
 			}
 			if err != nil {
-				err := utils.LavaFormatError("invalid Verification on provider startup", err, utils.Attribute{Key: "Addons", Value: addons}, utils.Attribute{Key: "verification", Value: verification.Name})
 				if verification.Severity == spectypes.ParseValue_Fail {
-					return err
+					return utils.LavaFormatError("invalid Verification on provider startup", err, utils.Attribute{Key: "Addons", Value: addons}, utils.Attribute{Key: "verification", Value: verification.Name})
```

Comment:
verifications can run not on provider startup, why did you add these attributes? aren't they added outside?

Reply:
they were there, i just moved the error print only if the is fail

Reply:
these attributes are not added anywhere